### PR TITLE
[App Service]: .NET SDK generated for API version "2021-03-01"

### PR DESCRIPTION
**Pull request by Kotasudhakarreddy**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by Kotasudhakarreddy** [comment]
@fengzhou-msft can you please help to succeed the build. We need to use this SDK to work on the breaking changes release.

**Comment by panchagnula** [comment]
@dingmeng-xue who can help us get some traction on this PR please? So we can update websites extension in PS to use the latest SDK as part of upcoming release? Thanks!

**Comment by dingmeng-xue** [comment]
Hi @ArthurMa1978 , could you help to assign team member to review it.

**Comment by Kotasudhakarreddy** [comment]
@ArthurMa1978 Any help is appreciated here.

**Comment by ArthurMa1978** [comment]
@fengzhou-msft  and @HarveyLink please help on this PR.

File: sdk/websites/Microsoft.Azure.Management.WebSites/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Web Sites Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Web Sites.")]
 
-[assembly: AssemblyVersion("3.1.2")]
-[assembly: AssemblyFileVersion("3.1.2")]
+[assembly: AssemblyVersion("3.2.0")]
```
Comment (ArthurMa1978) [inline sdk/websites/Microsoft.Azure.Management.WebSites/src/Properties/AssemblyInfo.cs:11]: Please increase the version to '4.0.0' as breaking changes came into this version

File: sdk/websites/Microsoft.Azure.Management.WebSites/src/Microsoft.Azure.Management.Websites.csproj [hunk 2]
```diff
@@ -7,9 +7,9 @@
 		<PackageId>Microsoft.Azure.Management.Websites</PackageId>
 		<Description>Provides website management capabilities for Microsoft Azure.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Websites</AssemblyName>
-		<Version>3.1.2</Version>
+		<Version>3.2.0</Version>
```
Comment (ArthurMa1978) [inline sdk/websites/Microsoft.Azure.Management.WebSites/src/Microsoft.Azure.Management.Websites.csproj:10]: Please increase the version to '4.0.0' as breaking changes came into this version

**Comment by ArthurMa1978** [comment]
@Kotasudhakarreddy 2 comments and please fix the ci issue

File: sdk/websites/Microsoft.Azure.Management.WebSites/src/Microsoft.Azure.Management.Websites.csproj (hunk 2 above)
Comment (Kotasudhakarreddy) [reply sdk/websites/Microsoft.Azure.Management.WebSites/src/Microsoft.Azure.Management.Websites.csproj]: Increased the version to `4.0.0`

**Comment by Kotasudhakarreddy** [comment]
> @Kotasudhakarreddy 2 comments and please fix the ci issue

@ArthurMa1978 I looking for help to fix CI issue. I am not sure what is causing the problem.

**Comment by HarveyLink** [comment]
> 

The pipeline shows there is a extra metadata folder exists, you should remove that folder.
[src/SDKs/_metadata/web_resource-manager.txt](https://github.com/Azure/azure-sdk-for-net/pull/28141/files#diff-79421a739e38fa3e1bb72a8101b564940defd2bbf2f49c8953f4ba4701ece6ae)

**Comment by Kotasudhakarreddy** [comment]
> > 
> 
> The pipeline shows there is a extra metadata folder exists, you should remove that folder. [src/SDKs/_metadata/web_resource-manager.txt](https://github.com/Azure/azure-sdk-for-net/pull/28141/files#diff-79421a739e38fa3e1bb72a8101b564940defd2bbf2f49c8953f4ba4701ece6ae)

@HarveyLink Thanks for the guidance! CI is passed now.

**Comment by Kotasudhakarreddy** [comment]
@ArthurMa1978 / @HarveyLink can you please review it.

**Review by panchagnula — APPROVED** [review]
Approved

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**